From 2f7baff1e206831945c83894fbc0ba5250178b5b Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 27 Mar 2014 14:45:53 +0000
Subject: [PATCH 54/72] Drop RedHatPlugin from procenv

The procenv package is not shipped in Red Hat distributions so
remove the RedHatPlugin tagging class from it.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/procenv.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sos/plugins/procenv.py b/sos/plugins/procenv.py
index e776e14..4367555 100644
--- a/sos/plugins/procenv.py
+++ b/sos/plugins/procenv.py
@@ -13,9 +13,9 @@
 ## along with this program; if not, write to the Free Software
 ## Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
-from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+from sos.plugins import Plugin, DebianPlugin, UbuntuPlugin
 
-class Procenv(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+class Procenv(Plugin, DebianPlugin, UbuntuPlugin):
     """Process environment.
     """
 
-- 
1.9.3

